health-services/project-factory/src/server/utils/targetUtils.ts

```
@@ -124,9 +130,75 @@
+    const localeFromTargetFile = await getLocaleFromCampaignFiles(targetFileId,tenantId);
+    const localizationMap = await getLocalizedMessagesHandlerViaLocale(localeFromTargetFile, tenantId);
+    const targetData = await getCodesTarget(campaignDetails, localizationMap);
+    const boundaryTree = await getBoundaryRelationshipDataFromCampaignDetails(campaignDetails);
+    if (targetData) {
+        mapTargets(
+            boundaryTree,
+            targetData
+        );
+        logger.debug(
+            "codesTargetMapping mapping :: " +
+            getFormattedStringForDebug(targetData)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)